### PR TITLE
replace wrong procedures + minor typos

### PR DESCRIPTION
**Pull request by ErrorRaffyline0**
Fixes info about tracking issues. Reapplies capitalization on mixed reality.

### CODE REVIEW

**Comment by ErrorRaffyline0** [comment]
@vtieto callback to #615

File: enthusiast-guide/headset-display.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ If you see "HoloLens Sensors" with a yellow exclamation mark in Device Manager,
```
Comment (ErrorRaffyline0) [inline enthusiast-guide/headset-display.md:100]: Everything in this paragraph is badly formatted in Preview Mode. I'm bad with formatting, could someone else fix it?

**Comment by ErrorRaffyline0** [comment]
Sorry if the reapplication of capitals on Mixed Reality was the wrong call, but seeing from the rest of the documentation it made more sense to keep it capitalized.

**Comment by ErrorRaffyline0** [comment]
Formatting still needs to be addressed, but I'm not gonna try that as I'd still be here a week after.

**Comment by ErrorRaffyline0** [comment]
> Sorry if the reapplication of capitals on Mixed Reality was the wrong call, but seeing from the rest of the documentation it made more sense to keep it capitalized.

New commit reverts this because I was stupid for changing this in the first place.

**Comment by ErrorRaffyline0** [comment]
@qianw211 plz fix formatting and merge

**Comment by ErrorRaffyline0** [comment]
Hey there @qianw211 thanks for the merge, but there is a part that I added that has poor formatting, I was hoping you could fix that since I'm very bad at markdown formatting...

**Comment by qianw211** [comment]
Hi @ErrorRaffyline0, thank you for updating the content.  I think the formatting looks fine:
https://learn.microsoft.com/en-us/windows/mixed-reality/enthusiast-guide/headset-display#the-images-shown-on-the-headsets-displays-appear-warped-or-incorrectly-scaled

We have limited writing resources, and this page is on our revamping work list.  Please look for updates in the near future.  Meanwhile, we're trying to minimize work done in this part of the documentation that's not the highest priority for us.  Thanks for your understanding.

**Pull request merged**